server/graphql/types/src/types/invoice_line.rs

```diff
@@ -158,6 +158,9 @@ impl InvoiceLineNode {
     pub async fn note(&self) -> &Option<String> {
         &self.row().note
     }
+    pub async fn return_reason_id(&self) -> &Option<String> {
+        &self.row().return_reason_id
+    }
```

Comment:
it's probably worth adding a loader for the return reason here? we'll display the return reasons on the detail view right?

Reply:
I think for now just in the next step of the line/item edit view. Loader + in detail view if we have time/budget, how does that sound ?

Reply:
Ah yep great 🙏